packages/internal/bridge/sdk/src/types/index.ts

```diff
@@ -110,6 +110,15 @@ export enum BridgeMethodsGasLimit { // @TODO test methods on chain and put corre
   MAP_TOKEN_DESTINATION = 200000,
   FINALISE_WITHDRAWAL = 200000,
   APPROVE_TOKEN = 55000,
+
+  DEPOSIT_SOURCE_ETH = 125000,
+  DEPOSIT_SOURCE_USDC = 153000,
```

Comment:
Maybe we should the USDC to 158K? I see [this](https://etherscan.io/tx/0x4ffca70b0f96c9b99d63bec790212bb2b94f2821297d7235c3996d77e95b880a) deposit transaction for 157.3K